src/webapp/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Button } from '@small-ads/ui';
+import styled from 'styled-components';
+import { useHistory, useRouteMatch } from 'react-router-dom';
+
+const HeaderStyles = styled.div`
+  position: sticky;
+  top: 0;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  max-width: 100%;
+  box-shadow: inset 0 -1px 0 0 rgba(0, 0, 0, 0.1);
+  min-height: 64px;
+`;
+
+const Header = styled.div`
+  width: 1048px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+  margin: 13.5px auto;
+`;
+
+export const Navbar = () => {
+  const history = useHistory();
+  const route = useRouteMatch('/login');
+  return (
+    <HeaderStyles>
+      <Header>
+        <Button
+          disabled={!!route?.isExact}
```

Comment:
```suggestion
          disabled={Boolean(route?.isExact)}
```

This way we can always know this is a boolean. Less error prone, mostly you can see the difference in most complex code but I am bringing this to your attention now even tho this might not be that complex. No need to apply this change just so you know this is an alternative which is more close to english language

Reply:
Great, great! Yes, please point out all those little things, I enjoy to learn them and I don't think it is annoying at all! I wanted to use something more robust than `!!` so I tried something with `typeof route?.isExact === undefined` but the linter complained.